tests/lib/test_teams.py

```
@@ -913,111 +926,108 @@ def test_transactions_structure(self, mock_fetch_html):
             assert hasattr(txn, 'country')
             assert hasattr(txn, 'position')
             assert hasattr(txn, 'reference_url')
-    
+
     def test_transactions_first_entry(self, mock_fetch_html):
```

Ruff (0.14.5)

Unused method argument: `mock_fetch_html`

(ARG002)